python/test/t_SobolIndicesAlgorithm_std.expout

```diff
@@ -361,3 +361,35 @@ Aggregated total order indices interval =  [0.481412, 0.505571]
  [ 0.00667425 0          ]]
 [[ 0          0.00667425 ]
  [ 0.00667425 0          ]]
+First order indices =  [0.906196,0.0868706,-0.00929482]
+Total order indices =  [0.911593,0.0977023,-1.62617e-11]
```

Comment:
@jschueller Will this tiny value be a problem on different platforms?

Reply:
no, it passes everywhere

Reply:
Oh, you were right, it fails on i386